src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -185,30 +185,31 @@
            [:div {:style {:marginTop "1em" :color (:exception-state style/colors) :textAlign "center"}}
             validation-error])
          [comps/ErrorViewer {:error submit-error}]
-         (flex/flex-box {:style {:marginTop 40}}
-                        (flex/flex-strut 80)
-                        flex/flex-spacer
-                        [comps/Button {:text "Previous"
-                                       :onClick (fn [_]
-                                                  (if-let [prev-page (peek (:pages-stack @state))]
-                                                    (swap! state #(-> %
-                                                                      (assoc :page-num prev-page)
-                                                                      (update :pages-stack pop)
-                                                                      (dissoc :validation-error)))))
-                                       :style {:width 80}
-                                       :disabled? (zero? page-num)}]
-                        (flex/flex-strut 27)
-                        [comps/Button {:text "Next"
-                                       :onClick #(react/call :next-page this)
-                                       :disabled? (> page-num (-> library-schema :wizard count))
-                                       :style {:width 80}}]
-                        flex/flex-spacer
-                        (let [save-permissions (and writer? curator?)
-                              last-page (> page-num (-> library-schema :wizard count))]
-                        [comps/Button {:text (if published? "Republish" "Submit")
-                                       :onClick #(react/call :submit this)
-                                       :disabled? (not (and save-permissions last-page))
-                                       :style {:width 80}}]))]]))
+         (flex/box
+          {:style {:marginTop 40}}
+          (flex/strut 80)
+          flex/spring
+          [comps/Button {:text "Previous"
+                         :onClick (fn [_]
+                                    (if-let [prev-page (peek (:pages-stack @state))]
+                                      (swap! state #(-> %
+                                                        (assoc :page-num prev-page)
+                                                        (update :pages-stack pop)
+                                                        (dissoc :validation-error)))))
+                         :style {:width 80}
+                         :disabled? (zero? page-num)}]
+          (flex/strut 27)
```

Comment:
Ugh, yeah.  That matches the existing OKCancelForm (which in turn comes from PixelMedia, again), which I unfortunately had to copy/paste instead of reusing.  Major target for future refactoring.